docs/docsite/rst/dev_guide/developing_collections_structure.rst

```diff
@@ -52,11 +52,28 @@ A collection must have a ``galaxy.yml`` file that contains the necessary informa
 .. _collections_doc_dir:
 
 docs directory
----------------
+--------------
+
+Use the ``docs`` folder to describe how to use the roles and plugins the collection provides, role requirements, and so on.
+
+For certified collections, Automation Hub displays documents written in markdown in the main ``docs`` directory.
+
+For community collections included in the Ansible PyPI package, docs.ansible.com displays documents written in reStructuredText (.rst) in a docsite/rst/ subdirectory. Define the structure of your extra documentation in ``docs/docsite/extra-docs.yml``:
+
+.. code-block:: yaml
+
+   ---
+   sections:
+   - title: Scenario Guide
+     toctree:
+       - scenario_guide
+
+The index page of the documentation for your collection displays the title you define in ``docs/docsite/extra-docs.yml`` with a link to your extra documentation. For an example, see the `community.docker collection repo <https://github.com/ansible-collections/community.docker/tree/main/docs/docsite>`_ and the `community.docker collection documentation <https://docs.ansible.com/ansible/latest/collections/community/docker/index.html>`_. 
```

Comment:
does it support subdirectories? If so, can we show that as an example? or if not, say no subdirectories?

Reply:
It should (https://github.com/ansible-community/antsibull/blob/main/antsibull/extra_docs.py#L54). But I haven't tried it, and I'm not aware of examples :)

Reply:
I think it does support subdirectories, but I don't think we have any examples that use that functionality yet. I'd vote for merging this, then updating it when we have some more complex examples.